src/Ylmish/Y.fs

```diff
@@ -38,192 +38,135 @@ module Y =
 
 [<RequireQualifiedAccess>]
 module Delta =
-
-    open Fable.Core.JS //
-
-    module ToAdaptive =
-        let private generate f start count =
-            let indexes = Index.generate f start count
-            let ops' = IndexListDelta.ofList indexes
-            let index' =
-                indexes
-                |> List.tryLast
-                |> Option.map fst
-                |> Option.defaultValue start
-            index', ops'
-
-        let folder getItem getCount ((index, ops, totalCount, currentCount) : Index * IndexListDelta<'b> * int * int) delta =
+    let inline applyYDelta
+        (getItem  : ^a -> int -> ^b)
+        (getLength: 'a -> int)
+        (delta: ResizeArray<Y.Delta<^a>>) (list: clist<^b>) : unit =
+        let rec loop i (delta: Y.Delta<^a> list) =

```

Comment:
Ha - I've hit that in the past, but https://github.com/fsharp/fslang-suggestions/issues/668 was marked as completed a few months ago and it's working fine for me in Visual Studio and on the command line (though I do also see this error in VS Code).

I do have .NET 7 installed, but because of the `global.json`, if I run `dotnet --version` on the command line it shows "6.0.405", so I would expect that to be what was used when building, so it's weird it works from the CLI for me (a mismatch between Visual Studio and the CLI experience isn't surprising in the least).

Ignoring all that - there's no need for SRTP here anyway - that's just a hangover from an earlier version of this code which used type constraints to specify members for getting items and lengths, which I abandoned when I realised I'd need `Count` for some types and `Length` for others - at which point I decided to just pass in the functions instead, given that type constraints are not easy to read.

I'll fix it.

Reply:
> I'll fix it.
Done.